### DISCUSSION TITLE
We can move page while entering commit message

### DISCUSSION THREAD

**Discussion by anatawa12** [community/community#60554] [Repositories]
### Select Topic Area

Bug

### Body

While we're editing file, github prevent from moving to other page as following
<img width="419" alt="image" src="https://github.com/community/community/assets/22656849/5c74da6d-e139-48cf-b0ca-f3b3fb353fba">

While we're editing file, github prevent from moving to other page as below.

However, when I started to write commit mesage, github doesn't prevent from moving to other page. I believe this is an bug.
<img width="503" alt="image" src="https://github.com/community/community/assets/22656849/f63f75e4-bee8-4c72-97bc-e7b8f45a11a2">

**Comment by jge162** [comment]
Seems like correct behavior.
